src/utils.coffee

```diff
@@ -17,7 +17,7 @@ device = require './device'
 
 exports.supervisorVersion = require('./lib/supervisor-version')
 
-configJson = require('/boot/config.json')
```

Comment:
I'd rather change this to reading the file and parsing it - require feels  more like a hack here. In any case this will all change in #465